templates/users/show.html

```diff
@@ -0,0 +1,34 @@
+{% extends "includes/base.html" %}
+{% load follower_tags %}
+{% block main_content %}
+<a href="{% url 'home' %}" style="float: right">Home</a>
+<h4>User Details</h4>
+<div>
+  <h6>First Name - {{ user.first_name }}</h6>
+  <h6>Last Name - {{ user.last_name }}</h6>
+  <h6>Followers - {{ user.followers.count }}</h6>
+  {% if request.user.id != user.id and not request.user|is_following:user %}
+    <a href="javascript:void(0)" onclick="follow('POST')">Follow</a>
+  {% elif request.user.id != user.id and request.user|is_following:user %}
+  <a href="javascript:void(0)" onclick="follow('DELETE')">Unfollow</a>
+  {% endif %}
+</div>
+{% endblock %}
+
+{% block specific_js %}
+  <script>
+    function follow(request_method) {
+      console.log("{{user.id}}");
+      fetch(`{% url 'users:ManageFollower' user.id %}`, {
```

Comment:
I told you to pass url and method from link. 

That way this ajax can be used for any  POST and DELETE

Reply:
okay my mistake let me make a new common function for all. please confirm for updating data